### PR TITLE
HealthChecks enabled but the plugins directory missing breaks (#2626)

### PR DESCRIPTION
**Pull request by MarekM25**
(cherry picked from commit e4eb3e1588dc09744260ea83a45803be97845099)

### CODE REVIEW

**Review by tkstanczak — APPROVED** [review]

**Comment by tkstanczak** [comment]
you may want to wait for a review from @matilote here

**Review by matilote — APPROVED** [review]

**Pull request merged**
